programs/margin-account/src/lib.rs

```diff
@@ -249,7 +248,7 @@ pub struct Withdraw<'info> {
 
 // TradeAMM takes the tokens that are in the margin account and executes a trade with them.
 #[derive(Accounts)]
-pub struct TradeAMM<'info> {
+pub struct TradeAmm<'info> {
```

Comment:
Do you think it's more idiomatic to have the Amm lower case?

Reply:
eh it's a Rust standard, one I don't really agree with and we can change config to allow, I was just doing least friction change